src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NonGCStaticsNode.cs

```diff
@@ -21,12 +22,19 @@ public class NonGCStaticsNode : ObjectNode, IExportableSymbolNode
     {
         private MetadataType _type;
         private NodeFactory _factory;
+        private List<PreInitFieldInfo> _sortedPreInitFields;
 
         public NonGCStaticsNode(MetadataType type, NodeFactory factory)
         {
             Debug.Assert(!type.IsCanonicalSubtype(CanonicalFormKind.Specific));
             _type = type;
             _factory = factory;
+            var preInitFieldInfos = PreInitFieldInfo.GetPreInitFieldInfos(_type, hasGCStaticBase: false);
```

Comment:
Couldn't think of anything better - I decided to go with the existing property name. At least there is no confusion.